scripts/zones/Jugner_Forest/IDs.lua

```diff
@@ -66,8 +66,13 @@ zones[xi.zone.JUGNER_FOREST] =
             [17203447] = 17203448,
         },
 
-        KING_ARTHRO = 17203216,
-        FRAELISSA   = 17203447,
+        KING_ARTHRO          = 17203216,
+        FRAELISSA            = 17203447,
+        DUESSA_START         = 17203449,
+        DUESSA_END           = 17203453,
+        METEORMAULER         = 17203414,
+        METEORMAULER_GUARD1  = 17203415,
```

Comment:
these are sequential, there isn't really a need to add the 2 helper mobs to the table. You can just `METEORMAULER +1` and `METEORMAULER +2` to refer to them.

Reply:
Adjusted the IDs for both Meteormauler and Fradubio's saplings.